setup.py

```diff
@@ -921,13 +921,6 @@ def my_new_compiler(**kw):
 
 
 class my_compiler(MSVCCompiler):
-    # Just one GUIDS.CPP and it gives trouble on mainwin too. Maybe I
-    # should just rename the file, but a case-only rename is likely to be
-    # worse!  This can probably go away once we kill the VS project files
-    # though, as we can just specify the lowercase name in the module def.
-    _cpp_extensions = MSVCCompiler._cpp_extensions + [".CPP"]
-    src_extensions = MSVCCompiler.src_extensions + [".CPP"]
```

Comment:
The only functional changes are in this file. Everything else is just using more consistent names for files that had a capitalized `.H` or `.CPP` extension